guidelines/sc/20/visual-presentation.html

```diff
@@ -13,5 +13,7 @@ <h4>Visual Presentation</h4>
      <li>Line spacing (leading) is at least space-and-a-half within paragraphs, and paragraph spacing is at least 1.5 times larger than the line spacing.</li>   
      <li>Text can be resized without assistive technology up to 200 percent in a way that does not require the user to scroll horizontally to read a line of text <a>on a full-screen window</a>.</li>   
   </ul>
+
+  <p class="note">The 'mechanism' for achieving the presentation can be provided by the user-agent, which can manipulate the content on behalf of the user. It is not a requirement for the author to provide the mechanism. Some writing systems have additional or different requirements, or do not customarily use features such as paragraph spacing in which case users might not use some or any of these requirements.</p>
```

Comment:
The option of user-agent mechanisms is explained in the first note of the "mechanism" definition. Repeating that text under this SC and no others gives the implication that this is somehow different from other criteria that use "mechanism", which shouldn't be the case. Better to keep the note focused on what is unique for this criterion in my opinion.

```suggestion
  <p class="note">Some writing systems have additional or different requirements, or do not customarily use features such as paragraph spacing in which case users might not use some or any of these requirements.</p>
```